src/api/clinical/lab.record/lab.record.controller.ts

```diff
@@ -137,8 +137,18 @@ export class LabRecordController extends BaseController {
     };
 
     //#endregion
+    private authorizeUser = async (request: express.Request, ownerUserId: uuid) => {
```

Comment:
I think authorizeOne does this job internally. No need to add authorize user method.

Reply:
Yes, Removed authorizeUser.